packages/abi-coder/src/Interface.ts

```diff
@@ -80,12 +96,14 @@ export class Interface<TAbi extends JsonAbi = JsonAbi> {
       );
     }
 
-    return AbiCoder.encode(this.jsonAbi, configurable.configurableType, value, {
-      encoding: this.encoding,
-    });
-  }
-
-  getTypeById(typeId: number) {
```

Comment:
Removed this method because the concept of a type isn't the same as previously. Also, you need the abi to instantiate an `Interface` class, so you can always search for the type in the abi directly.